lib/TPP/DefaultTppPasses.cpp

```diff
@@ -273,9 +273,8 @@ struct TppMappingPass : public TppMappingBase<TppMappingPass>,
     // mess up tensor producer-consumer chains used for analysis in the
     // following passes.
     pm.addPass(createPropagatePackUnPackPass());
-    pm.addPass(createCanonicalizerPass());
     pm.addPass(createConstantFoldPackPass());
-    pm.addPass(createCanonicalizerPass());
+    pm.addPass(createSimplifyAndCanonicalizePackPass());
```

Comment:
So, IIUC, this canonicalizes the packing and propagation of all operations (matmul, conv, etc) done so far?

Reply:
Yes, it runs canonicalization for `tensor.pack` and `tensor.unpack` plus some simplification. Added more doc on the td file.

Reply:
Thanks! LGTM.